newrelic-integration-e2e/main.go

```diff
@@ -15,15 +15,17 @@ const (
 	flagVerboseMode = "verbose_mode"
 	flagLicenseKey  = "license_key"
 	flagAgentDir    = "agent_dir"
+	flagRootDir     = "root_dir"
 )
 
 //go:embed resources/docker-compose.yml.tmpl
 var dockerComposeTemplate string
```

Comment:
as linter states, this is not used, but it is weird since it has a //go:embed directive

Reply:
fixed